Doc/library/os.rst

```diff
@@ -3462,18 +3470,23 @@ written in Python, such as a mail server's external command delivery program.
 
    .. versionadded:: 3.7
 
+   .. availability:: Unix. *setsid* parameter requires :c:data:
+   `POSIX_SPAWN_POSIX_SPAWN_SETSID` or :c:data:`POSIX_SPAWN_SETSID_NP` constant.
```

Comment:
Did you check is this doc compiles properly? I'm not sure about the Sphinx syntax. Check with "cd Doc; make venv; make html".

Reply:
The use of this :c:data: wouldnt compile. I decided to remove the sentence after unix.. Any suggestions otherwise.